target/xilinx/constraints/variables.tcl

```diff
@@ -0,0 +1,15 @@
+# Copyright 2022 ETH Zurich and University of Bologna.
+# Solderpad Hardware License, Version 0.51, see LICENSE for details.
+# SPDX-License-Identifier: SHL-0.51
+#
+# Cyril Koenig <cykoenig@iis.ee.ethz.ch>
+
+# Global variables are to be avoided in Vivado
+namespace eval ::params {
+  # 10 MHz (max) JTAG clock
+  variable JTAG_TCK 100.0
+  # UART speed is at most 5 Mb/s
+  variable UART_IO_SPEED 200.0
+  # Host on clk_50
+  variable SOC_TCK 20
```

Comment:
Ah I see, are we linking to this one? Is this our global source of truth? In that case, is there a need to inizialize them in the `xdc` and `tcl` constraints?